install/kubernetes/mesh-expansion.yaml

```diff
@@ -80,7 +81,29 @@ metadata:
 spec:
   type: LoadBalancer
   ports:
-  - port: 8060
+  - name: grpc-citadel
+    port: 8060
     protocol: TCP
+  - name: http-monitoring
+    port: 9093
   selector:
     istio: citadel
+---
```

Comment:
Note this file is no longer recommended - we use Gateways now, to reduce the number of LoadBalancers ( which are pretty expensive resources, with limits/quotas in some vendors). 
We're also now strongly recommending that all mesh expansion uses MTLS - we no longer ship insecure mesh expansion in the new installer.  Since mesh ex may in many cases be on separate network, clear text is too risky.
Is it possible to configure the zipkin cluster to use MTLS ? The Gateway can terminate it if needed.

Reply:
Thanks @costinm , Will try the Istio gateway with MTLS locally.